recipe/meta.yaml

```diff
@@ -102,15 +112,15 @@ build:
 
 outputs:
   - name: kernel-headers_{{ cross_target_platform }}
-    version: {{ kernel_headers_version }}
+    version: {{ kh_version }}
```

Comment:
Let's keep the name here

Reply:
It just gets very long if we use that prefix consistently... 🤷